wrapper/src/main/java/software/amazon/jdbc/wrapper/ConnectionWrapper.java

```diff
@@ -106,7 +111,8 @@ protected ConnectionWrapper(
       throw new IllegalArgumentException("url");
     }
 
-    init(props, connectionPluginManager, pluginService, hostListProviderService, pluginManagerService);
+    init(props, connectionPluginManager, pluginService, hostListProviderService,
```

Comment:
NIT: are those changes based on a checkstyle? it didn't seem that the line was that long to require a break

Reply:
I think my editor was using the wrong line limit setting and I never noticed.